docs/hooks/fields.mdx

```diff
@@ -80,7 +80,13 @@ The following arguments are provided to all Field Hooks:
 
 ### beforeValidate
 
-Runs before the `update` operation. This hook allows you to pre-process or format field data before it undergoes validation.
+Runs before the `create` and `update` operations. This hook allows you to add or format data before the incoming data is validated server-side.
+
+Please do note that this does not run before the client-side validation. If you added a `validate` function, this would be the lifecycle:
```

Comment:
Client-side validation will only run if you
- Have a custom field component where a hard-coded validate function is provided
- Manually render the respective Field component somewhere and provide `validate` as an arg

I would clarify this - otherwise, this gives the impression that the `validate` function will always run on the client if it has been provided in the Payload Config.